carla/models/catalog/catalog.py

```diff
@@ -49,10 +47,14 @@ def __init__(
         super().__init__(data)
         self._backend = backend
 
+        # TODO: Add logic for feature_order depending on upcoming different catalog models
+        self._catalog = load_catalog(data.catalog_file, data.name)
         if self._backend == "pytorch":
             ext = "pt"
+            self._feature_input_order = self._catalog["feature_input_order_bin"]
```

Comment:
Instead of loading the catalog here again, can't we make the feature orders part of the DataCatalog class and then just access the object property here?

E.g. 
```
self._feature_input_order = data.feature_order_bin
```

Reply:
You're right, an additional yaml for mlmodels would be less confusing, as you proposed in the next comment.
I'll change this in the next commit.